ingestion_tools/infra/db_import-v0.0.2.wdl

```diff
@@ -0,0 +1,114 @@
+version 1.0
+
+task cryoet_data_dbimport_workflow {
+    input {
+        String docker_image_id
+        String aws_region
+        String s3_bucket
+        String https_prefix
+        String environment
+        String flags
+    }
+
+    command <<<
+        set -euxo pipefail
+        export PYTHONUNBUFFERED=1
+        python --version 1>&2
+        ls -l 1>&2
+        original_dir=$(pwd)
+        cd /usr/src/app/ingestion_tools/scripts
+        set +x
+        POSTGRES_URL=$(aws secretsmanager get-secret-value --secret-id ~{environment}/db_uri | jq -r .SecretString | jq -r .db_uri)
+        echo python db_import.py load ~{s3_bucket} ~{https_prefix} POSTGRES_URL ~{flags} 1>&2
+        python db_import.py load ~{s3_bucket} ~{https_prefix} $POSTGRES_URL ~{flags} 1>&2
+        cd $original_dir
+        echo "done" > done.txt
+    >>>
+
+    runtime {
+        docker: docker_image_id
+    }
+    output {
+        String done = read_string("done.txt")
+    }
+}
+
+task cryoet_data_dbimport_v2_workflow {
+    input {
+        String done
+        String docker_image_id = "apiv2-x86:latest"
+        String aws_region
+        String s3_bucket
+        String https_prefix
+        String environment
+        String scrape_flags
+        String flags
+    }
+
+    command <<<
+        set -euxo pipefail
+        export PYTHONUNBUFFERED=1
+        python --version 1>&2
+        ls -l 1>&2
+        pwd 1>&2
+        set +x
+        apt install -y curl unzip jq
+        curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && unzip -q awscliv2.zip && ./aws/install && rm -rf ./aws
+        pip install psycopg2-binary
+        POSTGRES_URL=$(aws secretsmanager get-secret-value --secret-id ~{environment}/v2_db_uri | jq -r .SecretString | jq -r .db_uri)
+        # TODO - Scrape ID's from the v1 api so we keep things in sync
+        pip install cryoet-data-portal==3.1.1
```

Comment:
Could we move this to the poetry dependencies?

Reply:
It's not a dependency for the API service... I'm on the fence right now about creating a separate Dockerfile for the ingestion image vs the API service image. Which is something I'll probably do in the next few weeks, since all of these installs in the batch jobs are getting silly.